person.rb

```diff
@@ -17,5 +19,9 @@ def can_use_services?
     of_age? || @parent_permission == true
   end
 
+  def validate_name
+    @corrector.correct_name(name)
```

Comment:
- [ ] I suggest you assign the corrected name to the `@name` instance to ensure the corrected name is being saved. It should look similar to this `@name = @corrector.correct_name(@name)`